invenio_search_ui/assets/semantic-ui/js/invenio_search_ui/components/Results.js

```diff
@@ -69,6 +69,25 @@ export const Results = ({ currentResultsState = {} }) => {
               />
             </Grid.Column>
           </Grid.Row>
+          <Grid.Row className="mobile only" verticalAlign="middle">
+            <Grid.Column width={16} textAlign="center">
+              <Pagination
+                options={{
+                  size: "mini",
```

Comment:
I think it is more problematic if the size of the buttons is small, expecially if you need to change the page on the mobile